app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h3> <%= "Contestants" %> </h3>
+
+<% @contestants.each do |contestant| %>
+  <p> <%= contestant.name %> </p>
+  <p><%= "Projects: " %>
+  <% contestant.projects.each do |project| %>
+    <%= "#{project.name}" %>
+    <% if contestant.projects.last != project %>
```

Comment:
Check out `.to_sentence` so you don't have to have this logic preventing a trailing comma!